src/topology/uniform_space/cauchy.lean

```diff
@@ -330,3 +344,119 @@ instance complete_of_compact {α : Type u} [uniform_space α] [compact_space α]
 lemma compact_of_totally_bounded_is_closed [complete_space α] {s : set α}
   (ht : totally_bounded s) (hc : is_closed s) : compact s :=
 (@compact_iff_totally_bounded_complete α _ s).2 ⟨ht, is_complete_of_is_closed hc⟩
+
+/-! *** Sequentially complete space
+
+In this section we prove that a uniform space is complete provided that it is sequentially complete
+(i.e., any Cauchy sequence converges) and its uniformity filter admits a countable generating set.
+In particular, this applies to (e)metric spaces, see file `topology/metric_space/cau_seq_filter`.
+
+More precisely, we assume that there is a sequence of entourages $$U_n$$ such that any other
+entourage includes one of $$U_n$$. Then any Cauchy filter $$f$$ generates a decreasing sequence of
+sets $$s_n ∈ f$$ such that $$s_n × s_n ⊆ U_n$$. Choose a sequence $$x_n∈s_n$$. It is easy to show
+that this is a Cauchy sequence. If this sequence converges to some $$a$$, then $$f ≤ 𝓝 a$$. -/
```

Comment:
I don't think this will appear the way you want. Headers are introduced with `###`, and I don't think `$$` is supported syntax. In another PR, @bryangingechen said the following:
In case it helps, you can preview how the docstrings will render using https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%221757%22

Reply:
The preview page for this particular PR is at https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=1761 (you can change the url parameter to the number of the PR).

`$$` (or `$`) for LaTeX might be supported in the future if we decide to support LaTeX via mathjax in [the rendered docs](https://leanprover-community.github.io/mathlib_docs/topology/uniform_space/cauchy.html), I don't know if we came to a consensus on this yet though (cc @robertylewis).

Reply:
Fixed, thank you!